### PR TITLE
SD JWT javascript 2024 Annual Review

### PR DESCRIPTION
**Pull request by lukasjhan**
Hi TAC members.

I wrote the brief information of the annual review.

Thank you.

### CODE REVIEW

File: docs/projects/reviews/2024/2024-sd-jwt-js-annual.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# YYYY PROJECT NAME
```
Comment (tkuhrt) [inline docs/projects/reviews/2024/2024-sd-jwt-js-annual.md:1]:
```suggestion
# 2024 SD-JWT JavaScript Annual Review
```

File: docs/projects/reviews/2024/2024-sd-jwt-js-annual.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - 12 [Contributors](https://github.com/openwallet-foundation-labs/sd-jwt-js/graphs/contributors) since project inception
+  - [178 PRs merged](https://github.com/openwallet-foundation-labs/sd-jwt-js/pulls?q=is%3Apr+is%3Aclosed)
+  - [Total 1.2M+ NPM Downloads](https://npm-stats-ecru.vercel.app/)
+
+## Maintainer Diversity
+
+- Ace Shim ([Github](https://github.com/pensivej))
+- Lukas Han ([Github](https://github.com/lukasjhan))
+- Mirko Mollik ([Github](https://github.com/cre8))
+- Brend ([Github](https://github.com/berendsliedrecht))
```
Comment (tkuhrt) [inline docs/projects/reviews/2024/2024-sd-jwt-js-annual.md:19]: This list does not seem to match the code owners file.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Minor changes requested. Looks good otherwise.

File: docs/projects/reviews/2024/2024-sd-jwt-js-annual.md (hunk 2 above)
Comment (TimoGlastra) [inline docs/projects/reviews/2024/2024-sd-jwt-js-annual.md:19]:
```suggestion
- Berend Sliedrecht ([Github](https://github.com/berendsliedrecht))
```

Comment (lukasjhan) [reply docs/projects/reviews/2024/2024-sd-jwt-js-annual.md:19]: Thank you

File: docs/projects/reviews/2024/2024-sd-jwt-js-annual.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - 12 [Contributors](https://github.com/openwallet-foundation-labs/sd-jwt-js/graphs/contributors) since project inception
+  - [178 PRs merged](https://github.com/openwallet-foundation-labs/sd-jwt-js/pulls?q=is%3Apr+is%3Aclosed)
+  - [Total 1.2M+ NPM Downloads](https://npm-stats-ecru.vercel.app/)
+
+## Maintainer Diversity
+
+- Ace Shim ([Github](https://github.com/pensivej))
+- Lukas Han ([Github](https://github.com/lukasjhan))
+- Mirko Mollik ([Github](https://github.com/cre8))
+- Berend Sliedrecht ([Github](https://github.com/berendsliedrecht))
```
Comment (tkuhrt) [inline docs/projects/reviews/2024/2024-sd-jwt-js-annual.md:19]:
```suggestion
- Berend Sliedrecht ([Github](https://github.com/berendsliedrecht))
- Mirko Mollik ([Github](https://github.com/cre8))
```

**Review by tkuhrt — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Review by skounis — APPROVED** [review]

**Comment by tkuhrt** [comment]
Unanimously approved at the December 11, 2024 meeting

**Pull request merged**